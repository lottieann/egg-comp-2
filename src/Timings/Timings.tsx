import * as React from 'react'

import styled from 'styled-components'

const Title = styled.h1``

export const Timings = () => {
  return <Title>Schedule</Title>
}
