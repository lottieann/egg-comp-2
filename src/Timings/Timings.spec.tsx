import * as React from 'react'

import 'jest-styled-components'

import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { Timings } from './Timings'

const setup = () => {
  return <Timings />
}

describe('Timings', () => {
  it('should render Timings correctly', () => {
    const { container } = render(setup())
    expect(container).toMatchSnapshot()
  })
})
