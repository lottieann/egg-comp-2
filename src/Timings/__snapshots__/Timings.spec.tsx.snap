// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timings should render Timings correctly 1`] = `
.c0 {
  margin-top: 20vh;
}

.c0 p {
  font-family: Helvetica;
  text-align: center;
}

<div>
  <div
    class="c0"
  >
    <p>
      Entries submitted by 6pm Good Friday, see
       
      <b>
        <a
          href="/submit"
        >
          submit page
        </a>
      </b>
      .
    </p>
    <p>
      Gallery and voting available from Easter Saturday AM.
    </p>
    <p>
      Voting closes 6pm Easter Saturday
    </p>
    <p>
      Winners to be announced on Zoom at Easter Sunday noon GMT.
    </p>
  </div>
</div>
`;
